egs/librispeech/asr/nnlm/local/huggingface_tokenizer.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+# Copyright (c)  2020  Xiaomi Corporation (author: Liyong Guo)
+# Apache 2.0
+
+# reference: https://huggingface.co/docs/tokenizers/python/latest/quicktour.html
+import argparse
+import logging
+import os
+import shutil
+from pathlib import Path
+from tokenizers import Tokenizer
```

Comment:
Could you add some documentation describing how the environment is set up?
 I assume that you have run `pip install tokenizers` beforehand.

Reply:
No problem. A Readme.md will be added.